db/migrate/20170103144759_encrypt_user_credentials.rb

```diff
@@ -0,0 +1,6 @@
+class EncryptUserCredentials < ActiveRecord::Migration[5.0]
+  def change
+      rename_column :user_credentials, :credential_value, :encrypted_credential_value
```

Comment:
I agree, we should not force the encryption on users and only run the "migration" when `APP_ENCRYPTION_PASSPHRASE` is set. We should also lock the database during the migration to avoid race conditions in the different processes.